### DISCUSSION TITLE
What kind of table engines can be backed up and restored

### DISCUSSION THREAD

**Discussion by honobi** [Altinity/clickhouse-backup#989] [Q&A] [answered]
I saw in the README that it states:
"Only MergeTree family table engines (more table types for clickhouse-server 22.7+ and USE_EMBEDDED_BACKUP_RESTORE=true)."

Under the conditions of clickhouse-server 22.7+ and USE_EMBEDDED_BACKUP_RESTORE=true, what other table engines, besides the MergeTree family, can be backed up and restored?

**Comment by Slach** [comment, marked as answer]
with USE_EMBEDDED_BACKUP_RESTORE - Log Family engines can be backuped
https://clickhouse.com/docs/en/engines/table-engines/log-family
